src/app/complex/complex-results/complex-list/complex-list.component.css

```diff
@@ -21,3 +22,12 @@
 .basket-icons {
   margin-left: 5px;
 }
+
+.predicted h3 {
+  font-family: "Roboto Mono", monospace;
```

Comment:
I'm not sure about the different fonts.

<img width="1024" alt="Screenshot 2024-08-16 at 08 35 21" src="https://github.com/user-attachments/assets/eb1fcd49-a0fe-4d63-9fca-face7652ae16">

Reply:
It's definitely arguable as a choice, basically I needed a mono font to have consistent char width to get precisely 2 lines max, and then Roboto is pretty close to what we have, but different enough to mark the difference (+ I found it funny to use Roboto for machine predictions ^^).

Of course we could style the font at least to have the same size and weight as the curated ones, but I do think it convey the idea that these are somehow weaker. 

I'm perfectly willing to remove this though, or at least to tweak it so that it blends better, if the majority of people don't like it :D